<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamInfo</class>
 <widget class="QDialog" name="TeamInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о команде</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblEmblem">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/images/images/stub.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTeamName">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>My Favorite Team
(pashazz)
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabTeam">
      <attribute name="title">
       <string>Состав</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QListWidget" name="lstStats">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>118</red>
               <green>118</green>
               <blue>117</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>В стартовом составе, матчей</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/images/tactic.png</normaloff>:/images/images/tactic.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>На замену, матчей</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/images/change.png</normaloff>:/images/images/change.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>Голы</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/images/ball.png</normaloff>:/images/images/ball.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>Желтые карточки</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/yellow</normaloff>:/images/yellow</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>Красные карточки</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/images/cards.png</normaloff>:/images/images/cards.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
          <property name="toolTip">
           <string>Травмы</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/images/traumas</normaloff>:/images/traumas</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblPlayers">
         <column>
          <property name="text">
           <string>#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Позиция</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResults">
      <attribute name="title">
       <string>Результаты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gbWin">
         <property name="title">
          <string>Выигрыши</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="lstWin"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbDraw">
         <property name="title">
          <string>Ничьи</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListWidget" name="lstDraw"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbLose">
         <property name="title">
          <string>Поражения</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="lstLose"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabX11">
      <attribute name="title">
       <string>Бомбардиры</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tblGoals">
         <column>
          <property name="text">
           <string>Игрок</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Голов</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabYellow">
      <attribute name="title">
       <string>Желтые карточки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tblYellow">
         <column>
          <property name="text">
           <string>Игрок</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Карточек</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRed">
      <attribute name="title">
       <string>Красные карточки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableWidget" name="tblRed">
         <column>
          <property name="text">
           <string>Игрок</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Карточек</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TeamInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TeamInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
